src/helm/benchmark/scenarios/custom_mcqa_scenario.py

```diff
@@ -0,0 +1,116 @@
+import os
+from typing import Dict, Optional, List
+
+from helm.common.codec import from_jsonl
+from .scenario import Scenario, Instance, TRAIN_SPLIT, VALID_SPLIT
+
+
+class CustomMCQAScenario(Scenario):
+    """
+    A Generic Multiple Choice Question Answering Scenario that a user can use to
+    benchmark a model on their own data.
+    Code is adapted from the MMLU scenario.
+
+    We prompt models using the following format
+
+        <input>                  # train
+        A. <reference>
+        B. <reference>
+        C. <reference>
+        D. <reference>
+        Answer: <A/B/C/D>
+
+        x N (N-shot)
+
+        <input>                  # test
+        A. <reference1>
+        B. <reference2>
+        C. <reference3>
+        D. <reference4>
+        Answer:
+
+    For example (from mmlu:anatomy), we have:
+
+        The pleura
+        A. have no sensory innervation.
+        B. are separated by a 2 mm space.
+        C. extend into the neck.
+        D. are composed of respiratory epithelium.
+        Answer: C
+
+        Which of the following terms describes the body's ability to maintain its normal state?
+        A. Anabolism
+        B. Catabolism
+        C. Tolerance
+        D. Homeostasis
+        Answer:
+
+    Target: D
+    """
+
+    name = ""  # Will be set by the user in constructor
+    description = ""  # Will be set by the user in constructor
+    tags = ["multiple_choice"]  # Will be concatenated with the user's tags in constructor
+
+    def __init__(
+        self,
+        eval_path: str,
+        name: str,
+        description: str,
+        train_path: Optional[str],
+        num_train_instances: int = 0,
+        tags: Optional[List[str]] = None,
```

Comment:
Don't need `name`, `description`, `train_path` and `tags`. Rename `eval_path` to just `path`.